tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -61,8 +61,8 @@ def check_default_qspecs(compression_model):
                                        num_bits=8,
                                        signedness_to_force=None,
                                        per_channel=False,
-                                       narrow_range=False,
-                                       half_range=True)
+                                       narrow_range=True,
+                                       half_range=False)
```

Comment:
Could you provide a short explanation for the changes?

Reply:
I modified this test for 'target_device': 'TRIAL', hence the changes (half_range is disabled, as expected, in this case)

Reply:
Please use 'target_device': 'CPU' for these tests.

Reply:
Done